Remotion/Web/Development.WebTesting.IntegrationTests/BrowserSessionTest.cs

```diff
@@ -49,6 +49,7 @@ public void DeleteAllCookies_DeletesAllCookies ()
       Assert.That (allCookiesAfterDelete.Count, Is.EqualTo (0));
     }
 
+    [Ignore ("The Selenium log endpoints for Chrome are currently not working. (RM-7278)")]

```

Comment:
check if RM-7278 can be done first. that wa, we can harden the code against the upgrade.

Reply:
Can't be done, unfortunately.